src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationsHearingResultAwardResourceIntTest.kt

```diff
@@ -0,0 +1,231 @@
+          .headers(setAuthorisation(roles = listOf("ROLE_NOMIS_ADJUDICATIONS")))
+          .contentType(MediaType.APPLICATION_JSON)
+          .body(BodyInserters.fromValue(aHearingResultAwardRequest()))
+          .exchange()
+          .expectStatus().isNotFound
+          .expectBody()
+          .jsonPath("developerMessage").isEqualTo("Adjudication party with adjudication number 88888 not found")
+      }
+
+      @Test
+      fun `will return 400 if sanction type not valid`() {
+        webTestClient.post()
+          .uri("/adjudications/adjudication-number/$existingAdjudicationNumber/awards")
+          .headers(setAuthorisation(roles = listOf("ROLE_NOMIS_ADJUDICATIONS")))
+          .contentType(MediaType.APPLICATION_JSON)
+          .body(BodyInserters.fromValue(aHearingResultAwardRequest(sanctionType = "madeup")))
+          .exchange()
+          .expectStatus().isBadRequest
+          .expectBody()
+          .jsonPath("developerMessage").isEqualTo("sanction type madeup not found")
+      }
+
+      @Test
+      fun `will return 400 if sanction status not valid`() {
+        webTestClient.post()
+          .uri("/adjudications/adjudication-number/$existingAdjudicationNumber/awards")
+          .headers(setAuthorisation(roles = listOf("ROLE_NOMIS_ADJUDICATIONS")))
+          .contentType(MediaType.APPLICATION_JSON)
+          .body(BodyInserters.fromValue(aHearingResultAwardRequest(sanctionStatus = "nope")))
+          .exchange()
+          .expectStatus().isBadRequest
+          .expectBody()
+          .jsonPath("developerMessage").isEqualTo("sanction status nope not found")
+      }
+    }
+
+    @Nested
+    inner class HappyPath {
+
+      @Test
+      fun `create an adjudication hearing result award`() {
+        val hearingId =
+          webTestClient.post()
+            .uri("/adjudications/adjudication-number/$existingAdjudicationNumber/awards")
+            .headers(setAuthorisation(roles = listOf("ROLE_NOMIS_ADJUDICATIONS")))
+            .contentType(MediaType.APPLICATION_JSON)
+            .body(
+              BodyInserters.fromValue(
+                aHearingResultAwardRequest(),
+              ),
+            )
+            .exchange()
+            .expectStatus().isOk
+        assertThat(hearingId).isNotNull
+
+        verify(telemetryClient).trackEvent(
+          eq("hearing-result-award-created"),
+          org.mockito.kotlin.check {
+            assertThat(it).containsEntry("bookingId", prisoner.latestBooking().bookingId.toString())
+            assertThat(it).containsEntry("hearingId", existingHearing.id.toString())
+            assertThat(it).containsEntry("adjudicationNumber", existingAdjudicationNumber.toString())
+            assertThat(it).containsEntry("resultSequence", "1")
+            assertThat(it).containsEntry("sanctionSequence", existingCharge.id.chargeSequence.toString())
+          },
+          isNull(),
+        )
+      }
```

Comment:
I think we could also do a repository read and check that data on the database is correct